From 588206de19c2f0e69e4bb80d3cc4fad3bb724e83 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Sat, 19 Nov 2011 13:24:15 +0100
Subject: [PATCH 18/18] configure: Don't add -lX11 to the generic LIBS

This stops timidity itself from depending on libX11 even when build
without any interfaces builtin which need libX11.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 configure.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index 97c35db..56d3c6f 100644
--- a/configure.in
+++ b/configure.in
@@ -374,7 +374,7 @@ if test "x$with_x" = xyes -a "x$have_x" = xyes; then
     CPPFLAGS="$CPPFLAGS -I$x_includes"
   fi
 
-  AC_CHECK_LIB(X11,XOpenDisplay)
+  AC_CHECK_LIB(X11,XOpenDisplay,AC_DEFINE([HAVE_LIBX11],[],[Has libX11]))
   AC_MSG_CHECKING(X11 version 6)
   AC_CACHE_VAL(timidity_cv_x11_version_6,
     AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <X11/Xlib.h>]], [[
-- 
1.7.7.3

